### DISCUSSION TITLE
What does `Time to review` stand for?

### DISCUSSION THREAD

**Discussion by aleksei-commmune** [AlexSim93/pull-request-analytics-action#38] [Q&A] [answered]
Is 'Time to review' of user1 time taken to review the PRs created by user1 or is it time taken by user1 to review PRs?

**Comment by AlexSim93** [comment, marked as answer]
User_1 opened PR at 2 pm. User_2 reviewed it at 4 pm. Result is not important here, so it might be approve, changes requested or commented status. When PR will be merged, action adds value(2 hours) to User_1 info. After it action calculates result according to chosen method(average, median, percentile).
